aip/general/0009.md

```diff
@@ -102,6 +102,14 @@ The version of an API or a group of APIs if they are defined
 together. An API Version is often represented by a string, such as "v1", and
 presents in API requests and Protocol Buffers package names.
 
+### Client
+
+A consumer of the API. Examples of clients include:
+
+- A user who consumes the API directly (e.g. via a REST client).
```

Comment:
A "user" is a person, not a client.

Code that calls the API directly

Reply:
Actor is a more general synonym for "user". Could say, "machine actor".

Reply:
I don't know if we have to introduced a synonym for "user".

I'll go ahead and introduce the term "user", just to clarify that we define "clients" as code-based ones, while "user" refers to the end-user.

Reply:
ptal, added the term "user".